### PR TITLE
release: 4.6.0

### PR DESCRIPTION
**Pull request by code-asher**
Once approved I will make a new release candidate.

### CODE REVIEW

**Comment by jsjoeio** [comment]
The branch needs to have a "v" before the version number right?

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,23 @@ Code v99.99.999
...
+Code v1.70.1
+
+### Changed
+
+- Updated Code to 1.70.1.
+
+### Added
+
+- Added a heartbeat to sockets. This should prevent them from getting closed by
+  reverse proxy timeouts when idle like NGINX's default 60 second timeout.
```
Comment (jsjoeio) [inline CHANGELOG.md:34]:
```suggestion
  reverse proxy timeouts when idle like NGINX's default 60-second timeout.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,23 @@ Code v99.99.999
...
+
+### Fixed
+
+- Fixed logout option appearing even when authentication is diasbled.
```
Comment (jsjoeio) [inline CHANGELOG.md:38]:
```suggestion
- Fixed logout option appearing even when authentication is disabled.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +20,23 @@ Code v99.99.999
 
 -->
 
+## [4.6.0](https://github.com/coder/code-server/releases/tag/v4.6.0) - TBD
```
Comment (jsjoeio) [inline CHANGELOG.md:23]:
```suggestion
## [4.6.0](https://github.com/coder/code-server/releases/tag/v4.6.0) - 2022-08-17
```

**Review by jsjoeio — CHANGES_REQUESTED** [review]
minor changes. next gonna download artifact and test locally

**Comment by code-asher** [comment]
Closing this PR in favor of https://github.com/coder/code-server/pull/5469 which has the `v`.

**Pull request closed without merging**